lib/index.js

```diff
@@ -88,7 +91,11 @@ MapboxGeocoder.prototype = {
     render: function(item) {
       var placeName = item.place_name.split(',');
       return '<div class="mapboxgl-ctrl-geocoder--suggestion"><div class="mapboxgl-ctrl-geocoder--suggestion-title">' + placeName[0]+ '</div><div class="mapboxgl-ctrl-geocoder--suggestion-address">' + placeName.splice(1, placeName.length).join(',') + '</div></div>';
-    }
+    },
+    popupRender: function(item) {
+      var placeName = item.place_name.split(',');
+      return '<div class="mapboxgl-ctrl-geocoder--suggestion popup-suggestion"><div class="mapboxgl-ctrl-geocoder--suggestion-title popup-suggestion-title">' + placeName[0] + '</div><div class="mapboxgl-ctrl-geocoder--suggestion-address popup-suggestion-address">' + placeName.splice(1, placeName.length).join(',') + '</div></div>'
```

Comment:
Where do we define these classes `popup-suggestion`?

This seems very similar to the `render` method above, when and where would I choose this over the marker render?

Reply:
The render function above is used for rendering the search results list underneath the search bar. In this case I use a similar pattern to allow developers to override the popup's html while having a default method initially set

Reply:
Got it, do we not need to define some default styling for `popup-suggestion`?

Reply:
There are default styles defined on things like `mapboxgl-ctrl-geocoder--suggestion`, I added the `popup-suggestion` class name so that developers can override styles on the popup box only